### PR TITLE
Profile questions

### PR DESCRIPTION
**Pull request by Erin-Mounts**
Fix display of birthdate, height

### CODE REVIEW

**Comment by Erin-Mounts** [comment]
requires https://github.com/Sage-Bionetworks/ResearchUXFactory-iOS/pull/54

File: BridgeAppSDK/SBAProfileSection.swift [hunk 1]
```diff
@@ -158,19 +158,101 @@ open class SBAProfileItemProfileTableItem: SBAProfileTableItemBase {
...
+    @objc(hkQuantityheightAsItemDetail:)
+    open func heightAsItemDetail(_ height: HKQuantity) -> String {
+        let heightInCm = height.doubleValue(for: HKUnit(from: .centimeter)) as NSNumber
+        return self.heightAsItemDetail(heightInCm)
+    }
+    
+    open func heightAsItemDetail(_ height: NSNumber) -> String {
+        var answerString: String = String(describing: height)
+        
+        let formatter = NumberFormatter()
```
Comment (syoung-smallwisdom) [inline BridgeAppSDK/SBAProfileSection.swift]: https://developer.apple.com/documentation/foundation/lengthformatter

File: BridgeAppSDK/SBAProfileSection.swift [hunk 2]
```diff
@@ -158,19 +158,101 @@ open class SBAProfileItemProfileTableItem: SBAProfileTableItemBase {
...
+                    return choices.find({ SBAObjectEquality($0.choiceValue, obj) })?.choiceText ?? String(describing: obj)
+                case .account(.profile):
+                    guard let options = surveyItem.items as? [String],
+                            options.count == 1
+                        else { return String(describing: obj) }
+                    switch options[0] {
+                    case "birthdate":
+                        guard let date = obj as? Date else { return String(describing: obj) }
+                        return self.dateAsItemDetail(date)
+                    case "height":
```
Comment (syoung-smallwisdom) [inline BridgeAppSDK/SBAProfileSection.swift]: `SBAProfileInfoOption` includes these key words

**Review by syoung-smallwisdom — APPROVED** [review]
Few nits. Otherwise, looks good

**Pull request merged**